products/compute/api.yaml

```diff
@@ -7476,6 +7486,147 @@ objects:
             values:
               - :RESTART_NODE_ON_ANY_SERVER
               - :RESTART_NODE_ON_MINIMAL_SERVERS
+  - !ruby/object:Api::Resource
+    name: 'PacketMirroring'
+    min_version: beta
+    base_url: projects/{{project}}/regions/{{region}}/packetMirrorings
+    update_verb: :PATCH
+    self_link: projects/{{project}}/regions/{{region}}/packetMirrorings/{{name}}
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Using Packet Mirroring': 'https://cloud.google.com/vpc/docs/using-packet-mirroring#creating'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/v1/packetMirroring'
+    description: |
+      Packet Mirroring mirrors traffic to and from particular VM instances.
+      You can use the collected traffic to help you detect security threats
+      and monitor application performance.
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: The name of the packet mirroring rule
+        required: true
+      - !ruby/object:Api::Type::String
+        name: description 
+        description: A human-readable description of the rule.
+        input: true
```

Comment:
This is forceNew?

Reply:
It is!  Descriptions go that way most of the time.  It's nuts.